apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx

```diff
@@ -81,7 +83,16 @@ const UtilityTabResults = forwardRef<HTMLDivElement, UtilityTabResultsProps>(
                   >
                     increase the statement timeout
                   </a>
-                  .
+                  , if your query take more than 50s to run, please{' '}
```

Comment:
```suggestion
                  . If your query take more than 50s to run, please{' '}
```

Reply:
hmm it feels odd to specify the 50s here - for e.g from a user's POV I wouldn't know how long my query will take

Reply:
Yeah, I just wanted to make it clear that if the query take more than 50s it'll timeout no matter what.